action-custom.py

```diff
@@ -12,7 +12,18 @@
 
 HERMES_HOST = "localhost"
 SECONDS_IN_A_DAY = 86400.0
-ARTIST_ENTITY_NAME = "snips/artist"
+
+ENTITIES = {
+    "artists": "snips/artist",
+    "tracks": "snips/song",
+    "playlists": "playlistNameFR",
```

Comment:
Is it a localized field ?

Reply:
I am not sure what you mean, could you elaborate? 
But what I know is that this is how we have the slot name defined for the assistant that was provided for the installation.